src/frontend/apps/e2e/__tests__/app-impress/doc-export.spec.ts

```diff
@@ -79,7 +79,7 @@ test.describe('Doc Export', () => {
       })
       .click();
 
-    await page
+    void page
```

Comment:
I didn't kno it is possible to do that. It is not the same to remove `await` without `void` ? What means the usage of `void` here ?